whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapService.java

```diff
@@ -245,6 +250,35 @@ public Response help(@Context final HttpServletRequest request) {
                 .build();
     }
 
+    @GET
+    @Produces({MediaType.APPLICATION_JSON, CONTENT_TYPE_RDAP_JSON})
+    @Path("/{objectType}/rirSearch1/{relation}/{key:.*}")
+    public Response relationSearch(
```

Comment:
Can we create a different controller for rirsearch1 or these two paths are two entangled to seperate ? I believe there would be future changes or addition to it